src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>HTML5</title>
+</head>
+<body>
+Hello world!
```

Comment:
don't forget about whitespaces
```suggestion
  Hello world!
```